src/App.tsx

```diff
@@ -1,14 +1,70 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { FilterCriteria, Select, Status } from './types/otherTypes';
+
+const filterTodos = (todos: Todo[], filter: FilterCriteria): Todo[] => {
+  const { status, searchInput } = filter;
```

Comment:
```suggestion
const filterTodos = (todos: Todo[], { status, searchInput }: FilterCriteria): Todo[] => {
```
destructure it directly